src/replication.c

```diff
@@ -1662,11 +1642,12 @@ void readSyncBulkPayload(connection *conn) {
         server.repl_diskless_load == REPL_DISKLESS_LOAD_SWAPDB)
     {
         /* Create a backup of server.db[] and initialize to empty
-         * dictionaries */
-        diskless_load_backup = disklessLoadMakeBackups();
+         * dictionaries and a backup of server.cluster->slots_to_keys
+         * and initialize to empty radix tree. */
```

Comment:
let's keep this comment generic, so that we don't need to update it whenever we backup more things.
```suggestion
         * dictionaries. */
```